{"name": "# C5440 Delete User", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5440_Delete_user.py\", line 8, in admns_n_usrs\n    context.app.main_page.admns_n_usrs()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 189, in admns_n_usrs\n    actual_text = wait.until(EC.presence_of_element_located((USER))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1600907248019, "stop": 1600907250616}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1600907250616, "stop": 1600907256270}, {"name": "Then Click on the Users button", "status": "passed", "start": 1600907256270, "stop": 1600907256410}, {"name": "Then Make sure there are at least two Admins and one User role in the list of users", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5440_Delete_user.py\", line 8, in admns_n_usrs\n    context.app.main_page.admns_n_usrs()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 189, in admns_n_usrs\n    actual_text = wait.until(EC.presence_of_element_located((USER))).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1600907256411, "stop": 1600907274942}, {"name": "Then Click on the Settings 'three vertical' dots from the rightmost side of the user", "status": "skipped", "start": 1600907277060, "stop": 1600907277060}, {"name": "Then Choose and click on 'Delete' from the dropdown menu", "status": "skipped", "start": 1600907277060, "stop": 1600907277060}, {"name": "Then Confirm action by clicking OK button on the pop-up dialog window", "status": "skipped", "start": 1600907277060, "stop": 1600907277060}, {"name": "Then Then Verify that there is for one user less after deleting", "status": "skipped", "start": 1600907277060, "stop": 1600907277060}], "start": 1600907242746, "stop": 1600907277060, "uuid": "26f83811-3aa2-4e03-bb05-2bcecb0941d5", "historyId": "052d21b30082a18d0bb410aa5def4f94", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5440 Delete User"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}